analytics/analytics/job_processor/__init__.py

```diff
@@ -109,7 +110,10 @@ def create_job_fact(
     )
 
 
-@shared_task(name="process_job")
+@shared_task(
+    name="process_job",
+    autoretry_for=(ReadTimeout,),
+)
```

Comment:
While i think you're aware the default value is 3 for `max_retries`, it might be good to encode that here so everyone is aware:

```suggestion
@shared_task(
    name="process_job",
    autoretry_for=(ReadTimeout,),
    max_retries=3,
)
```